docs/upgrade-app.md

```diff
@@ -0,0 +1,13 @@
+---
+id: upgrade-app
+title: Upgrading App to Latest Version of React Native Windows
+---
+
+## Steps to Upgrade An App to New React Native Windows Version
+1. In the root of your project,  run `npx react-native-windows-init --version 0.XX.X --overwrite`.
```

Comment:
Note that this will not remove any files that were removed from the new template

Reply:
This will also stamp over the project GUID, which we probably would like to avoid

Reply:
Sounds good will add these callouts!

Reply:
I thought this was going to point to using upgrade helper? we should encourage automation rather than manually upgrading wherever possible (since it's error prone)

Reply:
Maybe I'm missing something here, but isn't upgrade helper the manual option? And then the RNW CLI is the automated option?

Reply:
I see - I think what's missing here is that it doesn't call out upgrading via the CLI, it starts with "run react-native-windows-init" so I think we need a bulletpoint or two before that describes the automated process?
also, if we have the data in upgrade helper, why wouldn't react-native upgrade remove files that were removed from the template?

Reply:
`react-native upgrade` (the automated one) is no longer working/maintained. Upgrade helper, and the manual diffs it shows, is the current solution.

Reply:
that's worrisome... the RN docs don't mention that: https://reactnative.dev/docs/upgrading
is there a bug tracking this / can we follow up?

Reply:
I was about to ask why there wasn't a back link to the RN upgrading docs. I knew that Windows didn't support the `upgrade` command, but based on the docs I assumed it worked for other platforms. If it's no longer supported definitely open an issue and maybe just post a PR the core docs to delete that [whole section](https://reactnative.dev/docs/upgrading#react-native-cli).

Even if the `upgrade` command part is wrong, I still think we should have a link back to [Upgrading to newer versions](https://reactnative.dev/docs/upgrading), as it has additional details about upgrade helper and such.

Reply:
I have a direct link to the upgrade helper section of the react native docs on this page. I can add a link to the upgrading page in general, but it might be confusing to the reader since there is upgrade info on that page that isn't applicable to RNW.